spec/integration/users_spec.rb

```diff
@@ -28,5 +29,19 @@
       ROLES.should include relationship.role
     end
   end
+  
+  it 'gets the investors that are tagged with the given tag' do
+    investors = client.user_tags(tag_id, :investor => 'by_residence')
+    
+    # Get the first investor user profile
+    user = client.get_user(investors.users[0].id)
+    
+    # Check his locations tags
+    location_match = false
+    user.locations.each do |location|
+      location_match = true if location.id == tag_id
+    end
```

Comment:
Not a big deal, but you could use `user.locations.any? { |loc| loc.id == tag_id }` here :smile: